### PR TITLE
Update syntax for 1.5.x nested queries

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/docs/products/databases/queries/+page.markdoc [hunk 1]
```diff
@@ -15,6 +15,7 @@ Appwrite SDKs provide a `Query` class to help you build queries. The `Query` cla
 | Example                 | Description                                                                                                           |
 |------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
 | `Query.select(["name", "title"])`  | Select which attributes should be returned from a document.                                                            |
+| `Query.and([Query.lessThan(5), Query.greaterThan(10)])`  | Returns document if it matches all of the nested sub-queries in the array passed in. |
 | `Query.or([Query.lessThan(5), Query.greaterThan(10)])`  | Returns document if it matches any of the nested sub-queries in the array passed in. |
```
Comment (abnegate) [inline src/routes/docs/products/databases/queries/+page.markdoc:19]: These need an attribute and they should be different for each query

File: src/routes/docs/apis/rest/+page.markdoc [hunk 2]
```diff
@@ -267,77 +262,96 @@ Appwrite's SDKs have a helper `ID.unique()` to generate unique IDs. When using A
...
+  "method": "cursorAfter",
+  "values": ["documentId"]
+}
+```
+
+## Query nesting {% #query-nesting %}
+Some Appwrite query methods, like `and` and `or`, allow you to nest queries. 
+When using Appwrite without an SDK, you can template your own JSON strings.
+
+In these cases, `attribute` is emtpy and `values` is an array of queries.
```
Comment (stnguyen90) [inline src/routes/docs/apis/rest/+page.markdoc:336]:
```suggestion
In these cases, `attribute` is empty and `values` is an array of queries.
```

File: src/routes/docs/apis/rest/+page.markdoc [hunk 3]
```diff
@@ -267,77 +262,96 @@ Appwrite's SDKs have a helper `ID.unique()` to generate unique IDs. When using A
 
 # Queries {% #queries %}
 
-Appwrite's SDKs provide a `Query` class to generate query strings. When using Appwrite without an SDK, you can template your own strings with the format below.
+Appwrite's SDKs provide a `Query` class to generate JSON query strings.
+When using Appwrite without an SDK, you can template your own JSON strings.
+You can discover the query methods available in the [Queries page.](/docs/products/databases/queries)
 
-Query strings are passed to Appwrite using the `queries` parameter. You can attach multiple query strings by including the array parameter multiple times in the query string: `queries[]="..."&queries[]="..."`
```
Comment (stnguyen90) [inline src/routes/docs/apis/rest/+page.markdoc:272]: The query string param part is still really helpful. Making it even more clear would actually be more helpful. For example, the query string **before** URL encoding would be

```
?queries[0]={"method":"equal","attribute":"name","values":["John"]}&queries[1]={"method":"limit","values":[6]}
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: src/routes/docs/products/databases/queries/+page.markdoc (hunk 1 above)
Comment (gewenyu99) [reply src/routes/docs/products/databases/queries/+page.markdoc]: Done

File: src/routes/docs/apis/rest/+page.markdoc (hunk 3 above)
Comment (gewenyu99) [reply src/routes/docs/apis/rest/+page.markdoc:272]: Added back

File: src/routes/docs/apis/rest/+page.markdoc (hunk 2 above)
Comment (gewenyu99) [reply src/routes/docs/apis/rest/+page.markdoc]: Done

File: src/routes/docs/products/databases/queries/+page.markdoc [hunk 4]
```diff
@@ -42,7 +43,8 @@ Appwrite SDKs provide a `Query` class to help you build queries. The `Query` cla
 | Example                 | Description                                                                                                           |
 |------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
 | `Query::select(["name", "title"])`  | Select which attributes should be returned from a document.                                                            |
-| `Query::or([Query.lessThan(5), Query.greaterThan(10)])`  | Returns document if it matches any of the nested sub-queries in the array passed in. |
+| `Query::or([Query::lessThan("size", 5), Query::greaterThan("size", 10)])`  | Returns document if it matches any of the nested sub-queries in the array passed in. |
+| `Query::and([Query::lessThan("size", 5), Query::greaterThan("size", 10)])`  | Returns document if it matches all of the nested sub-queries in the array passed in. |
```
[the remark this answers is not in the document]
Comment (gewenyu99) [reply src/routes/docs/products/databases/queries/+page.markdoc]: LOL fixed

**Review by abnegate — APPROVED** [review]

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**